moveit_planners/pilz_industrial_motion_planner/src/trajectory_functions.cpp

```diff
@@ -111,12 +120,12 @@ bool pilz_industrial_motion_planner::computePoseIK(const planning_scene::Plannin
                        timeout);
 }
 
-bool pilz_industrial_motion_planner::computeLinkFK(const moveit::core::RobotModelConstPtr& robot_model,
+bool pilz_industrial_motion_planner::computeLinkFK(const planning_scene::PlanningSceneConstPtr& scene,
```

Comment:
Do you really need the full PS or is the RobotState enough? If so, use the latter only.

Reply:
`RobotState` should be enough. Will do that now. I wonder if it's also worth removing [this](https://github.com/ros-planning/moveit/blob/9f8f6bbe0627619bd79928edd5f4d21047690255/moveit_planners/pilz_industrial_motion_planner/src/trajectory_functions.cpp#L134C3-L134C31) line, since we immediately set the joint values afterwards

Reply:
Done :+1: